content/en/developers/dogstatsd/data_types.md

```diff
@@ -242,7 +242,7 @@ def login(self, user_id)
 end
 ```
 
-Learn more about the [Histogram type in the Metrics documentation][7].
+Learn more about the [Sets type in the Metrics documentation][8].
```

Comment:
Link [8] goes to `/rates`. This link needs to point to `/sets'

Reply:
my brain is mush